config/test/affine.yaml

```diff
@@ -3,7 +3,7 @@ train:
   backbone:
     name: "global"
     num_channel_initial: 1
-    extract_levels: [0, 1, 2, 3, 4]
+    depth: 4
```

Comment:
does this modify back compatibility? because extract levels didn't need to be sequential before, this change suggests that all up to depth 4 is extracted?

Reply:
I've updated the back-compatibility parser, and also the model will calculate depth based on `extract_levels` in case depth is not provided.

So now we are offering more options, without impacting previous behaviors.